ansible_collections/arista/avd/docs/containers/overview.md

```diff
@@ -0,0 +1,92 @@
+<!--
+  ~ Copyright (c) 2023 Arista Networks, Inc.
+  ~ Use of this source code is governed by the Apache License 2.0
+  ~ that can be found in the LICENSE file.
+  -->
+
+# AVD Dev Containers Overview
+
+> WARNING: AVD Dev Container are currently in the early preview phase and not officially supported for any production cases.
+> They can be still very useful for development and testing. Please report any issues and optimization suggestions.
+
+## Why Dev Containers
+
+Building a functional Ansible environment can be a challenging task. A manually created Ansible environment can be broken due to number of reasons:
+
+- Problems with Python and Ansible versions
+- Broken dependencies
+- Interpreter path issues
+- Inconsistent Ansible configuration files
+- etc.
+
+Containers can provide a simple and portable way to build a functional and portable Ansible environment. [Dev Containers](https://containers.dev) bring that to the next level by simplifying container build process and extending the functionality. The [Dev Container Specification](https://github.com/devcontainers/spec) was started by Microsoft and has strong community support. Essentially Dev Containers are powered by following:
+
+- [Prebuilt images](https://github.com/devcontainers/images)
+- [Features](https://containers.dev/features)
+
+Dev containers can be run locally or remotely, in a private or public cloud, in a variety of [supporting tools and editors](https://containers.dev/supporting).
+
+## AVD Dev Container Variants
```

Comment:
Should we have an explanation of the naming convention? Like

`ghcr.io/aristanetworks/ansible-avd/<variant>:python<version>-avd-<avd version>`

Reply:
Certainly at a later point. Now right now, as a lot of changes expected at the start. Naming convention is not yet fixed.

Reply:
Understood.